docs/api-gateway-v2/03-04-migration.md

```diff
@@ -180,18 +97,48 @@ rules:
         - "http://dex-service.kyma-system.svc.cluster.local:5556/keys"
         - "http://auth-service.kyma-system.svc.cluster.local:5556/keys"
 ```
-The value for the last **rules.path** parameter is a regular expression with a negative lookahead. Its purpose is to exclude the paths already handled by other configuration settings from the `/.*` path to avoid two different configurations for a single path. 
-However, if the same `excludedPaths` element is present throughout the authentication settings, a particular path doesn't require any authentication. In that case, create a rule with `handler: allow`, so that you don't have to exclude the path using a negative lookahead.
+>**NOTE:** Path definitions in the APIRule CR cannot overlap, otherwise the requests to such paths are rejected. 
 
-  </details>
-</div>
+The **spec.rules** parameter corresponds to **spec.authentication** defined in the Api CR where you can specify a [list multiple JWT token issuers](https://kyma-project.io/docs/components/api-gateway/#details-security-specify-multiple-jwt-token-issuers) to allow secured access and use the **triggerRule.excludedPaths** parameter to exclude paths that don't require any authentication. 
 
+If you want to exclude paths in APIRule definition, for example to avoid two different configurations for a single path, you can use a regular expression with a negative lookahead. In the example, such an expression is used to in the last path exclude the paths already handled by other configuration settings from the `/.*` path. 
```

Comment:
Just to be sure. Shouldn't this part:
`[...] used to in the last path exclude [...]`
be `[...] used in the last path to exclude [...]` instead? Or was that intended?